### DISCUSSION TITLE
How to (pre-)fetch data in parallel

### DISCUSSION THREAD

**Discussion by Philipp-M** [rustwasm/gloo#489] [Q&A] [answered]
I was not sure whether this fits more into ideas, as I don't think this is yet supported.

I'm thinking about the equivalent of this javascript:

```js
await Promise.all([fetch(..), fetch(..)])
```

**Comment by ranile** [comment, marked as answer]
You'd do it the same way as how you'd run multiple futures concurrently elsewhere. [`futures::future::join`](https://docs.rs/futures/latest/futures/future/fn.join.html) is way of doing this

**Reply by Philipp-M** [reply]
Oh, yeah, good point, I somehow missed this, thanks, that sounds like a good solution.
